manifests/baseconfig/network/ifupdown.pp

```diff
@@ -0,0 +1,116 @@
+# Configure networking with ifupdown
+class profile::baseconfig::network::ifupdown (Hash $nics) {
+  $dns_servers = lookup('profile::dns::nameservers', {
+    'default_value' => undef,
+  })
+  $dns_search = lookup('profile::dns::searchdomain', {
+    'default_value' => undef
```

Comment:
Komma på enden av L7. Da blir den lik L4; og det er mindre sjans for tabber når det legges til en ny linje i hashmappet senere.